test.cpp

```
@@ -0,0 +1,48 @@
+    if (p == NULL) {
+        *p = 5;
+    }
+}
+
+int divide(int numerator, int denominator) {
+    return numerator / denominator;
+}
+
+int main() {
+    int uninitialized;
+    int result = uninitialized + 7;
```

Cppcheck (2.20.0)

[error] Uninitialized variable

(uninitvar)